### PR TITLE
LF-253: iframe integration script

### PR DESCRIPTION
**Pull request by chybisov**
iframe integration script gets new functionality including drawer mode 💥 

image
image

### CODE REVIEW

File: public/widget.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
+(function () {
+  const host = 'https://li.finance'
```
Comment (mathiasmoeller) [inline public/widget.js:2]: Do we want to make this configurable as well to allow e.g. local testing or staging?

File: public/widget.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+      toToken: null,
+      fromAmount: null
+    },
+  };
+  const drawerSettings = {
+    activeClass: 'is-active',
+    visibleClass: 'is-visible',
+    targetAttribute: 'data-drawer-target',
+    triggerAttribute: 'data-drawer-trigger',
+    closeAttribute: 'data-drawer-close',
```
Comment (mathiasmoeller) [inline public/widget.js:20]: The drawer gets injected directly, and is not inside an iframe right? Could general class names like `is-active` clash with user classes?

**Review by mathiasmoeller — COMMENTED** [review]
Looks good!

I guess we want to have this in JavaScript so we don't have to compile the TS?

The possible routes below the swap field and the resulting horizontal scrolling isn't the prettiest thing I know 😅  What's the plan here? Do we wait for the new design before we touch anything here? Could it be a quick fix to show the possible routes below each other?

**Comment by chybisov** [comment]
@mathiasmoeller Yeah, TS is not used there mainly because of how the current CRA config works. It can't compile files that are not in the /src folder. I'll figure this out for redesigned widget, not satisfied with that as well. This js file is also not minimized and obfuscated, which I plan to fix in a new one.

Regarding the quick fix for horizontal scroll, I agree it doesn't look lovely, we definitely can fix this, but I'm not sure we should spend time on this now. What do you think?

File: public/widget.js (hunk 1 above)
Comment (chybisov) [reply public/widget.js:2]: The same issue as with TS we don't have access to env variables here. If you have any suggestions, I would be happy to change this. 😄

File: public/widget.js (hunk 2 above)
Comment (chybisov) [reply public/widget.js:20]: Nice catch! 💪  I'll fix this.

File: public/widget.js (hunk 1 above)
Comment (mathiasmoeller) [reply public/widget.js:2]: Hmm yeah I get it. Would it make sense to pass it as a configuration parameter? I guess "users" should always use the production version so it doesn't make too much sense?
Let's just leave it for now I think.

**Review by mathiasmoeller — APPROVED** [review]
Yeah let's get a first version of the Widget released. We can still refine it in future iterations. Especially since the whole frontend part gets rebuild it probably doesn't make sense to spend too much time on that now 👍 
In the new setup we can also take into account proper compiling of the TS code and minification etc.

File: public/widget.js (hunk 1 above)
Comment (chybisov) [reply public/widget.js:2]: Yep, agree. Let's move forward.

**Pull request merged**